File: 백준/Silver/2012. 등수 매기기/등수 매기기.py
import sys

input = sys.stdin.readline

n = int(input())
visited = [i for i in range(1,1+n)]
nums = sorted([int(input()) for _ in range(n)])

result =0
for i in range(n):
    result +=abs(nums[i]-visited[i])

print(result)

File: 백준/Silver/2012. 등수 매기기/등수 매기기.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n")

const n = Number(input[0])
const visited = Array.from({ length: n }, (_, i) => i + 1)

const numbers = input
  .slice(1)
  .map(Number)
  .sort((a, b) => a - b)
let cnt = 0
for (let i = 0; i < n; i += 1) {
  cnt += Math.abs(numbers[i] - visited[i])
}

console.log(cnt)
